lib/components/tooltip.js

```diff
@@ -127,6 +134,38 @@ const addNpmLink = (data) => {
   return npm;
 };
 
+const addAtomLink = (data) => {
+  if (!data || !data.name) {
+    return undefined;
+  }
+
+  const dirlink = new Directory(atom.workspace.getActiveTextEditor().buffer.file.getParent().path+"/node_modules/"+data.name)
+  const existDirlink = dirlink.existsSync()
+
+  const icon = el('i');
+  setAttr(icon, {
+    className: 'icon icon-file-directory atom-icon',
```

Comment:
nice! I like the fallback. I'll add support for https://github.com/Yummygum/city-lights-icons-atom when this is merged as well 👍